sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_training_client.py

```diff
@@ -366,11 +358,10 @@ def get_copy_authorization(self, resource_id, resource_region, **kwargs):
     @distributed_trace
     def begin_copy_model(
         self,
-        model_id,  # type: str
-        target,  # type: Dict
-        **kwargs  # type: Any
-    ):
-        # type: (...) -> LROPoller[CustomFormModelInfo]
+        model_id: str,
+        target: dict,
```

Comment:
same comment here

Reply:
Is it considered breaking to be more specific with this dict type on this client?? 🤔

Reply:
Updated it to be more specific 👍